src/components/Theme/style.css

```diff
@@ -94,16 +94,6 @@
   height: 64*@unit;
 }
 
-.actions {
```

Comment:
Isn't this class shared with other components?

Reply:
no